### PR TITLE
feat: add LatestGasPrice component

### LINKED ISSUE #421: Display current blob gas price
Feature request by Ori:

> I love blobscan, but what I'm really interested in is the blob gas price. Can you add to the "Network: Mainnet..." line below the banner the latest blobgas cost, now that there is actually one that isn't always 1 wei?

### PR DESCRIPTION
**Pull request by luis-herasme**
### Checklist

- [ ] My change requires a documentation update, and I have done it.
- [ ] I have added tests to cover my changes.
- [x] I have filled out the description and linked the related issues.

### Description
Add the `LatestGasPrice` component to the `ExplorerDetails` component to display the latest gas price. This component fetches the latest gas price data from the API and renders it in the UI.

### Related Issue
Closes #421 

### Screenshots:
image

### CODE REVIEW

File: apps/web/src/components/GasPrice.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      <div>
+        <span className="font-bold">Blob gas price: </span>
+        {data.blobGasPrice.toString()}
+        <span className="test-sm opacity-80"> wei</span>
+      </div>
+    </div>
+  );
+}
+
+function GasSVG() {
```
Comment (PJColombo) [inline apps/web/src/components/GasPrice.tsx:22]: Let's do the same than described [here](https://github.com/Blobscan/blobscan/pull/464#discussion_r1700386719)

File: apps/web/src/components/ExplorerDetails.tsx [hunk 2]
```diff
@@ -55,6 +56,7 @@ export function ExplorerDetails() {
 
   return (
     <div className="sm:fle flex w-full flex-wrap items-center justify-center gap-2 align-middle text-xs text-contentSecondary-light dark:text-contentSecondary-dark sm:h-4 sm:justify-start">
+      <LatestGasPrice />
```
Comment (PJColombo) [inline apps/web/src/components/ExplorerDetails.tsx:59]: Can we refactor this to include it as part of the `explorerDetailsItems` array?

Comment (PJColombo) [reply apps/web/src/components/ExplorerDetails.tsx:59]: Take a look at how it's done for the "Last synced slot" item

File: packages/api/src/routers/block/getGasPrice.ts
Comment (PJColombo) [inline packages/api/src/routers/block/getGasPrice.ts:1]: Let’s refactor this procedure to return the latest block and get the gas price from there to maintain consistency with the rest of the procedures.

Use the `findLatest` method in the `block` Prisma model.

**Review by PJColombo — CHANGES_REQUESTED** [review]
Great job!

I’ve left some comments requesting a few refactoring changes.

File: apps/web/src/components/ExplorerDetails.tsx (hunk 2 above)
Comment (luis-herasme) [reply apps/web/src/components/ExplorerDetails.tsx]: Done

File: apps/web/src/components/ExplorerDetails.tsx [hunk 3]
```diff
@@ -35,7 +36,19 @@ function ExplorerDetailsItem({
...
   const { data: blobStoragesState } = api.blobStoragesState.getState.useQuery();
+  const { data: latestBlock } = api.block.getLatestBlock.useQuery();
+
   const explorerDetailsItems: ExplorerDetailsItemProps[] = [
+    {
+      name: "Blob gas price",
+      icon: <Gas className="h-4 w-4" />,
+      value: latestBlock ? (
+        <div>
+          {latestBlock.blobGasPrice.toString()}
```
Comment (PJColombo) [inline apps/web/src/components/ExplorerDetails.tsx:47]: We have a [component](https://github.com/Blobscan/blobscan/blob/next/apps/web/src/components/Displays/EtherUnitDisplay.tsx) to display ether units. Let's use it and fix the amount to `Gwei`

File: apps/web/src/components/ExplorerDetails.tsx [hunk 4]
```diff
@@ -35,7 +36,19 @@ function ExplorerDetailsItem({
...
+          <span className="text-xs opacity-80"> wei</span>
+        </div>
+      ) : undefined,
+    },
```
Comment (PJColombo) [inline apps/web/src/components/ExplorerDetails.tsx:51]: Let's display this as the final element

**Review by PJColombo — APPROVED** [review]
I added a couple of last comments I forgot to include in the first revision

Feel free to squash/merge this once they get resolved 

Also, don't forget to add the changeset

File: apps/web/src/components/ExplorerDetails.tsx (hunk 4 above)
Comment (luis-herasme) [reply apps/web/src/components/ExplorerDetails.tsx]: Done

**Pull request merged**